arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -209,21 +236,50 @@
               "$ref": "#/definitions/MediaService"
             }
           }
-        ]
-      },
-      "delete": {
-        "summary": "Delete a Media Service.",
-        "operationId": "MediaService_Delete",
+        ],
         "responses": {
-          "200": {
-            "description": "OK"
+          "201": {
+            "description": "Success. Returns the new Media Services account.",
+            "schema": {
+              "$ref": "#/definitions/MediaService"
+            },
+            "examples" : {
+              "application/json": {
+                "name": "contosomedia",
+                "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Media/mediaservices/contosomedia",
+                "type": "Microsoft.Media/mediaservices",
+                "location": "West US",
+                "tags":null,
+                "properties":{  
+                  "apiEndpoints":[  
+                    {  
+                      "endpoint": "https://wamsbayclus001rest-hs.cloudapp.net/api/",
+                      "majorVersion": "2"
+                    }
+                  ],
+                  "storageAccounts":[  
+                    {  
+                      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Storage/storageAccounts/contosomedia",
+                      "isPrimary": true
+                    }
+                  ]
+                }
+              }  
+            }
           },
           "default": {
-            "description": "Detailed error information.",
+            "description": "Create Media Services failed.",

```

Comment:
Would this be better as "Create Media Services failures.", or "Create Media Services that failed", or "Create Media Services operation failed." ?